tests/integration/records/views/test_views_literature.py

```diff
@@ -542,3 +547,96 @@ def test_literature_citation_annual_summary_for_many_records(
     response = api_client.get(f"literature/facets/?{urlencode(request_param)}")
     expected_response = {"value": {"2013": 2, "2012": 1, "2010": 1}}
     assert response.json["aggregations"]["citations_by_year"] == expected_response
+
+
+def test_literature_search_user_does_not_get_fermilab_collection(
+    api_client, db, es_clear, create_record, datadir
+):
+    data = {
+        "$schema": "http://localhost:5000/schemas/records/hep.json",
+        "_collections": ["Fermilab"],
+        "control_number": 666,
+        "document_type": ["article"],
+        "titles": [{"title": "Partner walk again seek job."}],
+    }
+
+    create_record("lit", data=data)
+
+    headers = {"Accept": "application/json"}
+    expected_status_code = 200
+
+    response = api_client.get("/literature", headers=headers)
+    response_status_code = response.status_code
+    response_data = json.loads(response.data)
+
+    assert response_data["hits"]["total"] == 0
+    assert expected_status_code == response_status_code
+
+
+def test_literature_search_cataloger_gets_fermilab_collection(
+    api_client, db, es_clear, create_record, datadir, create_user
+):
+    data = {
+        "$schema": "http://localhost:5000/schemas/records/hep.json",
+        "_collections": ["Fermilab"],
+        "control_number": 666,
+        "document_type": ["article"],
+        "titles": [{"title": "Partner walk again seek job."}],
+    }
+    user = create_user(role=Roles.cataloger.value)
+    login_user_via_session(api_client, email=user.email)
+
+    create_record("lit", data=data)
+
+    headers = {"Accept": "application/json"}
+    expected_status_code = 200
+    expected_data = {
+        "$schema": "http://localhost:5000/schemas/records/hep.json",
+        "_collections": ["Fermilab"],
+        "control_number": 666,
+        "document_type": ["article"],
+        "titles": [{"title": "Partner walk again seek job."}],
+        "citation_count": 0,
+        "citations_by_year": [],
+    }
+
+    response = api_client.get("/literature", headers=headers)
+    response_status_code = response.status_code
+    response_data = json.loads(response.data)
+    assert response_data["hits"]["total"] == 1
+
+    response_data_metadata = response_data["hits"]["hits"][0]["metadata"]
+
+    assert expected_status_code == response_status_code
+    assert expected_data == response_data_metadata
+
+    print(api_client.session_transaction)
+
+
+def test_literature_search_permissions(
+    api_client, db, es_clear, create_record, datadir, create_user
+):
+    def _logout():
+        current_user.authenticated = False
+        current_user.roles = []
+
+    create_record("lit", data={"_collections": ["Fermilab"]})
+    create_record("lit")
+
+    headers = {"Accept": "application/json"}
+    response = api_client.get("/literature", headers=headers)
+    response_data = json.loads(response.data)
+    assert response_data["hits"]["total"] == 1
```

Comment:
shall we check the control number here? Just to make sure the response is the ```_collections: ["Literature"]``` etc